app/design/adminhtml/default/default/template/sales/order/view/info.phtml

```diff
@@ -98,7 +98,17 @@ $orderStoreDate = $this->formatDate($_order->getCreatedAtStoreDate(), 'medium',
                 <td class="label"><label><?php echo Mage::helper('sales')->__('%s / %s rate:', $_order->getOrderCurrencyCode(), $_order->getBaseCurrencyCode()) ?></label></td>
                 <td class="value"><strong><?php echo $_order->getBaseToOrderRate() ?></strong></td>
             </tr>
-            <?php endif ?>
+            <?php endif;
+            
+            foreach($this->getAdditionalInfo() as $label => $htmlFragment){
+                ?>
+                <tr>
+                    <td class="label"><label><?=$this->escapeHtml($label) ?></label></td>
+                    <td class="value"><?=$htmlFragment ?></td>
```

Comment:
As we directly use this as a html fragment here, I would like to make the array structure a bit more complex, so its already clear on populating the array, the value is used in an unescaped way.

so filling it like this:

```
$array['label']  = [
    "htmlFragment" => "<span>my html</span>",
];
```

Iam worried else its to easy people populate it with unexcaped data, which opens it up to xss attacks.